mesa/time.py

```diff
@@ -31,6 +31,15 @@
 from mesa.agent import Agent, AgentSet
 from mesa.model import Model
 
+# Throw module-level deprecation warning
+warnings.warn(
```

Comment:
Since time.py is imported into `__init__.py`, this will throw a warning all the time `mesa` is imported. This will be too noisy.

Reply:
That's a good catch, and I agree. the fix is to remove it from the overall init then?

Reply:
One option is to add the warning to every classes `__init__`.

Reply:
Would the warning in the Basescheduler work for now? Since all other schedulers use it.

Reply:
Moved.

Reply:
Doesn’t really seem to solve it.

Reply:
It should go into the init itself. A class is parsed on import so currently the error is thrown at that moment already. In the init the error gets thrown when initializing the class.

Reply:
Done, and I checked that all other schedulers call the Baseclass init, which is the case.